src/common.rs

```diff
@@ -3213,8 +3384,7 @@ impl User {
 /// ```
 #[derive(PartialEq, Eq, PartialOrd, Ord, Debug)]
 pub struct Group {
-    pub(crate) id: Gid,
```

Comment:
Any reason for this change? `Gid` is already generated based on the OS so what required for a `GroupInner` type to be created?